src/web/src/components/Posts/Post.tsx

```diff
@@ -98,6 +101,25 @@ const useStyles = makeStyles((theme: Theme) =>
         marginLeft: '.1em',
       },
     },
+    postCount: {
+      marginRight: '16px',
```

Comment:
For this one, I use 16px because of the padding left and right of the title.
![image](https://user-images.githubusercontent.com/51014333/143983259-1d18461e-fdbc-4813-b151-4ef92383db6f.png)